atomix/cluster/src/main/java/io/atomix/cluster/messaging/impl/NettyUnicastService.java

```diff
@@ -150,6 +144,24 @@ protected void channelRead0(
     return bind(serverBootstrap);
   }
 
+  private void handleReceivedPacket(final DatagramPacket packet) {
+    final int preambleReceived = packet.content().readInt();
+    if (preambleReceived != preamble) {
+      log.warn("Received unicast message from outside of the cluster. Ignoring the message");
```

Comment:
Could we add more context here like where does it came from etc.

Reply:
Added sender address.

Reply:
Thanks :+1: